generator/uscore/metadata_extractor.rb

```diff
@@ -559,6 +559,11 @@ def add_special_cases(metadata)
           (path.start_with? 'component.value') && (!path.include? '[x]') && (path != 'component.valueQuantity')
         end
 
+        # remove carrierAIDC and carrierHRF from implantable device must supports
+        implantable_device_sequence = metadata[:sequences].find { |sequence| sequence[:profile] == 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-implantable-device' }
```

Comment:
I think we should have a general solution, like a predefined list of "must-supported element group" which fall into the same pattern, ie all the elements in such group are marked as must-supported but only one of element is required to be supported.
Soon, there will be more of similar groups.

Reply:
I think we should keep this simple solution until we need the more complicated one.  Right now this is the only either-or must support.  When v3.2 lands, I think we can address the many either-or must supports.  Part of my motivation here is that I would like a stable solution that addresses today's program certification problem today (which this covers), and I'm concerned that making a stable more general solution will take a little while.